patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -782,15 +782,15 @@
  
          private ToolMaterial(int p_i1874_3_, int p_i1874_4_, float p_i1874_5_, float p_i1874_6_, int p_i1874_7_)
          {
-@@ -1034,6 +1739,7 @@
+@@ -1034,6 +1735,7 @@
              return this.field_78008_j;
          }
  
 +        @Deprecated // Use getRepairItemStack below
          public Item func_150995_f()
          {
              if (this == WOOD)
-@@ -1057,5 +1763,21 @@
```

Comment:
You probably should not include this patch change.

Reply:
Ah ha, you located The Thing that made all the line numbers change. I was looking for that and couldn't find it. I know I didn't touch the file myself, I assumed that it was still "needed" in some capacity but I couldn't figure out why. I can remove it (I presume that the decompile process added that annotation?).

Reply:
yeah genPatches sometimes changes patches for files you didn't actually change - in that case simply revert it using `git checkout -- <offending patch>` before committing.

Reply:
I keep the local repo and the dev environment separate from each other and use a file diff utility to move changes. It lets me firewall things so I don't have to use git commands like that often/at all. For fixing it, I just have to checkout the trunk, copy the patch file "somewhere safe" then checkout my branch, and copy it back (or in this case, I opened it in Notepad++ and then told it not to reload from disk, then resaved the file).